<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAnnotationOptionsPanelWidget</class>
 <widget class="QWidget" name="CreateAnnotationOptionsPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="saveAnnotationsInnerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Choose a file to store the annotations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbExistingTable">
        <property name="text">
         <string>Existing table:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cbExistingTable">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToMinimumContentsLength</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbBrowseExistingTable">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../U2Gui.qrc">
            <normaloff>:/core/images/load_selected_documents.png</normaloff>:/core/images/load_selected_documents.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCreateNewTable">
        <property name="text">
         <string>Create new table:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="leNewTablePath">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbBrowseNewTable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="rbUseAutoTable">
        <property name="text">
         <string>Use auto-annotations table</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="annotationParametersInnerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblGroupName">
        <property name="text">
         <string>Group name:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="leGroupName">
          <property name="placeholderText">
           <string>auto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbSelectGroupName">
          <property name="toolTip">
           <string>Predefined group names</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../U2Gui.qrc">
            <normaloff>:/core/images/group_auto.png</normaloff>:/core/images/group_auto.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblAnnotationType">
        <property name="text">
         <string>Annotation type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbAnnotationType">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAnnotationName">
        <property name="text">
         <string>Annotation name:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="leAnnotationName">
          <property name="maxLength">
           <number>100</number>
          </property>
          <property name="placeholderText">
           <string>by type</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblLocation">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="leLocation">
          <property name="toolTip">
           <string>Annotation location in GenBank format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbDoComplement">
          <property name="toolTip">
           <string>Add/remove complement flag</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../U2Gui.qrc">
            <normaloff>:/core/images/do_complement.png</normaloff>:/core/images/do_complement.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblDescription">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDescription"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chbUsePatternNames">
        <property name="toolTip">
         <string>Use names of patterns as annotations names. In case the patterns are in FASTA format</string>
        </property>
        <property name="text">
         <string>Use pattern name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../U2Gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rbCreateNewTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>leNewTablePath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCreateNewTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>tbBrowseNewTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbExistingTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbExistingTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbExistingTable</sender>
   <signal>toggled(bool)</signal>
   <receiver>tbBrowseExistingTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
